tools/apigee-envoy-quickstart/extension/setup-external-access.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+init() {
+  export CLUSTER_CTX="gke_${PROJECT_ID}_${CLUSTER_LOCATION}_${CLUSTER_NAME}"
+  export ASM_GATEWAYS_NAMESPACE="istio-ingressgateway"
+  export USE_GKE_GCLOUD_AUTH_PLUGIN=True
+  export TOKEN_TYPE="Bearer"
+  export MGMT_HOST="https://apigee.googleapis.com"
```

Comment:
Variable is used only once and never meant to change? Maybe just put the hostname directly in the curl?